package.json

```diff
@@ -33,7 +33,7 @@
   "bin": "cli.js",
   "scripts": {
     "test": "make lint test",
-    "prepublish": "make dist"
+    "prepublish": "make NIS= dist"
```

Comment:
It is a variable, not a make goal. This argument changes the `NIS` variable from its default setting (a file name) to the empty string. So it essentially removes `NIS` as a dependency from all the rules mentioning it, thus avoiding an infinite recursion. Should I add a comment, or are comments a bad idea in a file that gets re-written by npm from time to time?